src/site/shared/containers/ItemNav/index.tsx

```diff
@@ -198,8 +202,31 @@ export const ItemNav = <D,>({ info, config, additionalData, onStart, onFinish, a
                                                    onStart && onStart();
                                                }
                                            }}>
-                                    <img src={`/${config.imgRoot}/${section.id}/${item.icon}`} alt={item.label} />
-                                    <br />
+
+
+                                    <div onMouseEnter={() => {
+                                             if (item.removable) {setHover(item.id)}
+                                         }}
```

Comment:
Can reduce this to `<div onMouseEnter={() => (item.removable && setHover(item.id))}`
and similarly for `onMouseLeave`